coralproject/talk
# Featured Notifications

## What does this PR do?

- Adds featured notifications ⭐️ 

## How do I test this PR?

1. Enable the plugin (server + client)
2. Have your settings enabled for "My comment is featured"
3. Post a comment.
4. Feature your comment.
5. Check your email!
